TAC/Meetings/2022/2022-03-10/CCC_TAC_Minutes-2022-03-10.md

```diff
@@ -0,0 +1,190 @@
+* **Declared project dependencies**: [Google Sheets](https://docs.google.com/spreadsheets/d/1UKnbbGWXYLjnPZsox3zmYo59nv3XSXjePfas5E2fER0/edit#gid=0)
+* **YouTube**: [youtube.confidentialcomputing.io](https://youtube.confidentialcomputing.io)
+* **LFX**: [lfx.linuxfoundation.org](https://lfx.linuxfoundation.org)
+* **Join the CCC**: [join.confidentialcomputing.io](https://join.confidentialcomputing.io)
+* **Contact the CCC**: [confidentialcomputing.io/contact-us/](https://confidentialcomputing.io/contact-us/)
+
+## Agenda and Minutes
+
+### Welcome
+
+Rep. Thaler opened the call at 7:05a PT. He welcomed the members of the TAC and reviewed the values of the CCC and the antitrust policy of the Linux Foundation.
+
+### Roll Call / Attendance
+
+#### Voting members
+
+* Dan Middleton (Intel)
+* Dave Thaler (Microsoft)
+* Eric Northup (Meta)
+* Lily Sturman (Red Hat)
+* Zhipeng (Howard) Huang (Huawei)
+* Thomas Fossati (Arm)
+
+#### Other attendees
+
+* Ben Fischer (Red Hat)
+* Brian Warner (Linux Foundation)
+* Eric Voit (Cisco)
+* Helen Lau (Linux Foundation)
+* Henk Birkholz ()
+* Kurt Taylor (Linux Foundation)
+* Naveen Cherukuri (Nvidia)
+* Mike Bursell (Profian)
+* Nick Vidal (Profian)
+* Penglin Yang (China Mobile)
+* Sneha Damle (R3)
+* Simon Johnson (Intel)
+* Steve Van Lare (Anjuna)
+* Xinxin Fan (IoTeX)
+
+### Meeting minute approvals
+
+#### Approved by voting members
+
+**RESOLVED:** That the minutes of the [December 16, 2021](../../2021/2021-12-16/TAC_Minutes-2021-12-16.pdf) meeting of the Technical Advisory Council meeting of the Confidential Computing Consortium as distributed to the members of the TAC in advance of this meeting are hereby adopted and approved.
+
+**RESOLVED:** That the minutes of the [February 10, 2022](https://github.com/confidential-computing/governance/pull/93/files) meeting of the Technical Advisory Council meeting of the Confidential Computing Consortium as distributed to the members of the TAC in advance of this meeting are hereby adopted and approved.
+
+**RESOLVED:** That the minutes of the * [December 16, 2021](../../2021/2021-12-16/TAC_Minutes-2021-12-16.pdf) meeting of the Technical Advisory Council meeting of the Confidential Computing Consortium as distributed to the members of the TAC in advance of this meeting are hereby adopted and approved.
+
+#### Pending approval
+
+* None
+
+### Open Action Items
+
+1. **[Stephen, Zongmin]** Recommend diversity and inclusion training to their projects and report back to the TAC on whether the maintainers or the contributors will be taking it and when the expected completion date is.
+    * Gramine reports they have taken it.
+    * Enarx reports they have taken it.
+
+1. **[Stephen Walli, Dan Middleton, Aeva Black]** Develop a CoC escalation process, to be presented to TAC and governing board.
+    * Dan Middleton has [opened a PR for review](https://github.com/confidential-computing/governance/pull/92)
```

Comment:
I think we decided this is a duplicate of the action below and that since we now have a PR we would track this in the PRs instead of as a separate action item.